### PR TITLE
Feat/share knowledge

### LINKED ISSUE #7: Paragraph on "We share knowledge"

### PR DESCRIPTION
**Pull request by garloff**
closes #7

### CODE REVIEW

File: MANIFESTO.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ The answer to this must be the collectivization of operational knowledge, just a
 From time to time, we can get mentally stuck and may need a sparring partner to think outside the box. Open Operations builds a community of practice to keep the barrier to entry low and create a thriving environment for comfortable exchange. Regardless of individual knowledge level, everyone contributes and enriches this community with their knowledge and individual views. Communities of practice connect through various channels serving both asynchronous and synchronous communication needs in the form of pair operating or on a larger scale to solve these challenges together. Therefore, we are building a network by encouraging organizations to foster and connect internal and external communities of practice.
 
 ## We share knowledge
-[...]
+Availability of knowledge and skilled engineers is the limiting factor for many organizations to adopt, leverage and successfully operate (complex) technology.
```
Comment (fkr) [inline MANIFESTO.md:14]: Imho we should not fall back to putting stuff in ( ). The manifest should be down to the point, short and easy to read.
Either it is part of the sentence or it is not. My suggestion:
"[...] and successfully operate complex technology".

File: MANIFESTO.md [hunk 2]
```diff
@@ -11,7 +11,9 @@ The answer to this must be the collectivization of operational knowledge, just a
...
+While the open source movement has been successful in collectivizing a lot of software, organizations much too often continue to protect operational knowledge like a holy grail. This must be overcome to allow engineers to learn more effectively and to mitigate the shortage of available experience.
```
Comment (fkr) [inline MANIFESTO.md:15]: So far - at least that was my understanding from the sessions - we emphasized on what to do better without blaming what is wrong. To follow the overall pattern of the manifesto, I'd see to rephrase the first sentence.

"The open source movement has been successful in collectivizing a lot of software, organizations need to follow the same pattern for operational knowledge.
This allows engineers to learn[...]"

**Review by fkr — CHANGES_REQUESTED** [review]
see inline comments.

**Comment by itrich** [comment]
Hi @maxwolfs,

I just want to be sure that you're aware of this PR. Hope that you both can somehow merge your results on this section. I'd propose to keep working on this branch and either push directly or open further PR against `feat/share-knowledge`.

Looking forward to meeting your tomorrow!

**Comment by maxwolfs** [comment]
> Hi @maxwolfs,
> 
> I just want to be sure that you're aware of this PR. Hope that you both can somehow merge your results on this section. I'd propose to keep working on this branch and either push directly or open further PR against `feat/share-knowledge`.
> 
> Looking forward to meeting your tomorrow!

I actually already had a branch open but not pushed yet. Therefore here my suggestion, where I mainly swapped the inner paragraph:

The availability of knowledge and skilled engineers is the limiting factor for many organizations to adopt, leverage, and successfully operate complex technology. For true self-determination in the ever-growing digital realm, we urgently need to build skills and share the gained knowledge freely and in unlimited ways also in the realm of operational knowledge — exactly the way we started with software code almost 40 years ago. We encourage organizations to collectivize operational knowledge and share it with the entire Open Operations community.

**Review by itrich — DISMISSED** [review]
LGTM!

File: MANIFESTO.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ The answer to this must be the collectivization of operational knowledge, just a
...
 
 ## We share knowledge
-[...]
+The availability of knowledge and skilled engineers is the limiting factor for many organizations to adopt, leverage, and successfully operate complex technology. For true self-determination in the ever-growing digital domain, we urgently need to build skills and share the gained knowledge freely and in unlimited ways also in the realm of operational knowledge — exactly the way we started with software code almost half a decade ago. We encourage organizations to collectivize operational knowledge and share it with the entire Open Operations community.
```
Comment (fkr) [inline MANIFESTO.md:14]: Last minor nit - this struck me a week ago, however I did not adhere to it, because I was not sure wether it is important or was just the moment. Now re-reading the paragraph I'm stumbling across it again, so I at least want to raise it, however this is by far no blocker:

"For true self-determination in the ever-growing digital domain, we urgently need to build skills[...]"

I would rephrase that and leave the urgently out there:

"For true self-determination in the ever-growing digital domain, we need to build skills[...]"

**Review by fkr — DISMISSED** [review]
Left a comment. however I don't see it as a blocker, please note it however (and see wether it reasonates with you the same).

**Review by itrich — APPROVED** [review]
LGTM! Let's merge this and iterate further.
